charts/datadog/templates/NOTES.txt

```diff
@@ -534,6 +534,25 @@ More information about this change: https://github.com/DataDog/helm-charts/pull/
 {{- end }}
 
 
+{{- if and (eq .Values.targetSystem "linux") (eq (include "should-add-host-path-for-os-release-paths" .) "false") .Values.datadog.sbom.host.enabled }}
```

Comment:
I think I should add another condition to check that `.Values.datadog.osReleasePath` is not empty. 
because even if `should-add-host-path-for-os-release-paths == "false"` it should work with value in `.Values.datadog.osReleasePath`

Reply:
I made the change in https://github.com/DataDog/helm-charts/pull/1611/commits/b111c4a89a8409d35422956860e7a79ce0aeb055